apps/coscrad-frontend/src/components/notes/note-detail.presenter.tsx

```diff
@@ -1,21 +1,8 @@
 import { INoteViewModel } from '@coscrad/api-interfaces';
 
-export const NoteDetailPresenter = ({
-    id,
-    note: text,
-    connectedResources: relatedResources,
-}: INoteViewModel) => (
+export const NoteDetailPresenter = ({ id, note: text }: INoteViewModel) => (
     <div>
         <h2>Note ({id})</h2>
         <p>{text}</p>
-
-        <h2>Connected Resources</h2>
-        <p>
-            {relatedResources.map((resource, index) => (
```

Comment:
There should be  a story to do this properly.

Reply:
It's [right here](https://www.pivotaltracker.com/story/show/183962233). I have tagged a comment in the code with a link.